spy_integration_test.go

```diff
@@ -9,24 +9,34 @@ import (
 )
 
 func TestExecuteIntegration(t *testing.T) {
-	var buff bytes.Buffer
-	want := "echo Hello\nHello\n"
-	err := spy.Execute(&buff, "echo Hello")
+	want := struct{ a, b string }{
```

Comment:
This feels less like a struct and more like one long, newline-separated string, doesn't it?